packages/devops/scripts/lambda/helpers/create_from_image.py

```diff
@@ -68,7 +68,7 @@ def create_tupaia_instance_from_image(
 ):
     deployment_type = 'tupaia'
     tupaia_server_iam_role_arn = 'arn:aws:iam::843218180240:instance-profile/TupaiaServerRole'
-    tupaia_subdomains = ['','admin','admin-api','api','config','export','mobile','psss','report-api','psss-api','entity-api','lesmis-api','lesmis', 'meditrak-api', 'data-table-api'] if setup_gateway else None
+    tupaia_subdomains = ['','admin','admin-api','api','config','export','mobile','psss','report-api','psss-api','entity-api','lesmis-api','lesmis', 'meditrak-api', 'data-table-api', 'tupaia-web-api', 'tupaia-web-rewrite'] if setup_gateway else None
```

Comment:
Separately, this has been uploaded to our AWS lambda, so all branches will now be reserving a `*-tupaia-web-api` and `*-tupaia-web-rewrite` url. But only after the `servers.conf` update will those urls actually point to anything

Reply:
Also, obviously `*-tupaia-web-rewrite` is a temporary value for the duration where we want _both_ versions of the frontend available at the same time